docs/source/coroutines.rst

```diff
@@ -273,6 +273,81 @@ It can be used to create a special type of generator function that can be iterat
 More details on this type of generator can be found in :pep:`525`.
 
 
+Calling blocking code
```

Comment:
```suggestion
.. _calling_blocking_code:

Calling blocking code
```
So that you can add a reference to this page from the docstrings of `bridge()` and `resume()`,
like in https://github.com/cocotb/cocotb/blob/3b9fee3ddbba491bd5b7267fe56989b3e6bcd95d/src/cocotb/triggers.py#L925C5-L925C48